src/FSharpLint.Core/Rules/Conventions/Binding/UselessBinding.fs

```diff
@@ -19,13 +19,13 @@ let private checkForUselessBinding (checkInfo:FSharpCheckFileResults option) pat
         
         let isNotMutable (symbol:FSharpSymbolUse) =
             match symbol.Symbol with
-            | :? FSharpMemberOrFunctionOrValue as v -> not v.IsMutable
+            | :? FSharpMemberOrFunctionOrValue as fSharpMemberOrFunctionOrValue -> not fSharpMemberOrFunctionOrValue.IsMutable
```

Comment:
@webwarrior-ws too long, let's rename to fsharpElement

Reply:
done